layers/meta-opentrons/recipes-robot/opentrons-robot-server/opentrons-robot-server.bb

```diff
@@ -19,7 +19,7 @@ PIPENV_APP_BUNDLE_PROJECT_ROOT = "${S}/robot-server"
 PIPENV_APP_BUNDLE_DIR = "/opt/opentrons-robot-server"
 PIPENV_APP_BUNDLE_USE_GLOBAL = "numpy systemd-python python-can wrapt pyzmq "
 PIPENV_APP_BUNDLE_STRIP_HASHES = "yes"
-PIPENV_APP_BUNDLE_EXTRAS = "./../hardware"
```

Comment:
was this delete on purpose?

Reply:
Yeah, that's actually in the package dependencies now so it was showing up twice